modules/plot/test/spec.coffee

```diff
@@ -1,8 +1,12 @@
+# Used to mimic an event call for a node
+fakeNodeEvent = (node, eventName) ->
+  if node?
+    (e) -> hx.select.getHexagonElementDataObject(node).eventEmitter?.emit((if eventName? and isNaN(eventName) then eventName else 'click'), e)
```

Comment:
Why is isNaN being used here ? Are you anticipating a number string here ?

Would it be clearer to pull out the hexagon event name to a variable?